### PR TITLE
per issue #195: corrected notes and call log header styles and increased height of no…

### PR DESCRIPTION
**Pull request by dfelsent**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Changes the call log heading to h2
* Changes the notes partial to h2
* Adjusts the size of the notes text area

It relates to the following issue #s: 
* Fixes #291

### CODE REVIEW

**Comment by mebates** [comment]
:purple_heart: Thanks Deb!

File: app/views/pregnancies/_call_log.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <section id="call_log" class="call-info tab-pane">
 	<div class="margin-bottom col-md-12">
-		<h1 class="border-bottom title">Call Log</h1>
+		<h2>Call Log</h2>
```
Comment (colinxfleming) [inline app/views/pregnancies/_call_log.html.erb]: :100:

**Comment by colinxfleming** [comment]
Here's how this checks out -- 

screen shot 2016-05-28 at 11 58 19 am

@mebates want to give this your stamp of approval and then hit the merge button?

**Comment by colinxfleming** [comment]
(Also, looks good to me! Thanks for pushing this work thru @dfelsent !)

**Comment by mebates** [comment]
Great! The box can still be wider, but the depth helps.

**Pull request merged**